docs/notebooks/getting_started/particles.ipynb

```diff
@@ -0,0 +1,903 @@
+   "source": [
+    "argon = Particle(\"Ar 0+\")\n",
+    "argon.ionize(inplace=True)\n",
+    "print(argon)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e5302bef",
+   "metadata": {},
+   "source": [
+    "## Custom particles"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4c8c2a63",
+   "metadata": {},
+   "source": [
+    "[CustomParticle]: ../../api/plasmapy.particles.particle_class.CustomParticle.rst\n",
+    "\n",
+    "Sometimes we want to use a particle with custom properties.  For that we can use [CustomParticle]."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "35b94505",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from astropy import constants as const\n",
+    "from astropy import units as u\n",
+    "\n",
+    "custom_particle = CustomParticle(9.27e-26 * u.kg, 13.6 * const.e.si, symbol=\"Fe 13.6+\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dfc19404",
+   "metadata": {},
+   "source": [
+    "[CustomParticle]: ../../api/plasmapy.particles.particle_class.CustomParticle.rst\n",
+    "[Particle]: ../../api/plasmapy.particles.particle_class.Particle.rst\n",
+    "\n",
+    "Many of the attributes of [CustomParticle] are the same as in [Particle]."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8a4b412b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "custom_particle.mass"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a2da2b3b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "custom_particle.charge"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ecb450f6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "custom_particle.symbol"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "21a992a9",
+   "metadata": {},
+   "source": [
+    "[nan]: https://numpy.org/doc/stable/reference/constants.html#numpy.nan\n",
+    "\n",
+    "If we do not include one of the physical quantities, it gets set to [nan] (not a number) in the appropriate units."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a31c421a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "CustomParticle(9.27e-26 * u.kg).charge"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f1ed1dc2",
+   "metadata": {},
+   "source": [
+    "## Molecules"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "72bb94e5",
+   "metadata": {},
+   "source": [
+    "[CustomParticle]: ../../api/plasmapy.particles.particle_class.CustomParticle.rst\n",
+    "[molecule]: ../../api/plasmapy.particles.particle_class.molecule.rst\n",
+    "\n",
+    "We can use [molecule] to create a [CustomParticle] based on a chemical formula. The first argument to [molecule] is a string that represents a chemical formula, except that the subscript numbers are not given as subscripts. For example, water is ``\"H2O\"``.  "
```

Comment:
Since this is a custom particle, presumably many of the functions + properties (nuclear reactions, etc.) are not usable here too?

Reply:
Correct. I've been meaning to refactor and expand `@particle_input` to allow for `CustomParticle` and `ParticleList` inputs for formulary functions, though hopefully I'll have a chance to address this sometime soon.